docs/requirements.txt

```diff
@@ -1,3 +1,6 @@
 mkdocs>=1.6.0
 mdx_truly_sane_lists>=1.3
 pymdown-extensions>=10.8.1
+mkdocs-material>=9.5.47
+mkdocs-git-revision-date-localized-plugin
+mkdocs-git-authors-plugin
```

Comment:
We always include explicit version ranges for dependencies.

This needs a trailing newline; make sure your editor is set up to respect our `.editorconfig` file.